VisualPinball.Unity/Documentation~/creators-guide/setup/installing-vpe.md

```diff
@@ -38,7 +38,7 @@ Once the editor has opened you can click away the HDRP Wizard that opens. You sh
 
 Click on *File -> New Scene* and select the *Basic Indoors (HDRP)* template. Save it in your *Assets/Scenes* folder as `TestTable.unity`.
 
-In this base scene there's a plane that will [Z-fight](https://en.wikipedia.org/wiki/Z-fighting) with our imported table later. Select it in the *Hierarchy* and move it down a little by setting the *Y Position* under *Transform* in the *Inspector* to `-0.1`. Your scene should now look like this:
+In this base scene there's a *Plane* that will [Z-fight](https://en.wikipedia.org/wiki/Z-fighting) with our imported table later. Select it in the *Hierarchy* ( left by default) and move it down a little by setting the *Y Position* under *Transform* in the *Inspector* (right by default) to `-0.1`. Your scene should now look like this:

```

Comment:
If it's not clear enough, I'd prefer linking to Unity's doc ([hierarchy](https://docs.unity3d.com/Manual/Hierarchy.html) / [inspector](https://docs.unity3d.com/Manual/UsingTheInspector.html)).